### ISSUE TITLE
Read write for reverse mount

### ISSUE THREAD

**Issue by syaifulnizamyahya** [vgough/encfs#242]
I use reverse mount as below

```
echo <password> | ENCFS6_CONFIG='/boot/acd_cli/config/.encfs6.xml' encfs -S --reverse -o rw -o allow_other -o uid=99 -o gid=100 /mnt/user/Amazon/local/ /mnt/user/Amazon/.local/
```

Unfortunately, .local folder is read only. How do I mount .local as read write? I want to move .local contents to a remote folder(in the cloud). I set my encryption to be file independent. Meaning, encrypted files can be placed anywhere and yet can still be decrypted.

**Comment by EldonMcGuinness** [comment]
I wish this was possible as in my use case it makes it a bit cumbersome.

**Comment by diskmaster23** [comment]
Did anyone ever answer this question? I am having the same issue.

**Comment by pcjonathan** [comment]
I too wish this was possible.

**Comment by rfjakob** [comment]
There seems to be a bug where the very first mount is read-only. Subsequent mounts are read-write.

```
$ encfs --version
	encfs version 1.9.1
$ mkdir a b
$ touch a/foo
$ encfs --reverse $PWD/a $PWD/b
	Creating new encrypted volume.
	Please choose from one of the following options:
	 enter "x" for expert configuration mode,
	 enter "p" for pre-configured paranoia mode,
	 anything else, or an empty line will select standard mode.
	?> 

	Standard configuration selected.

	Configuration finished.  The filesystem to be created has
	the following properties:
	Filesystem cipher: "ssl/aes", version 3:0:2
	Filename encoding: "nameio/block", version 4:0:2
	Key Size: 192 bits
	Block Size: 1024 bytes
	File holes passed through to ciphertext.

	Now you will need to enter a password for your filesystem.
	You will need to remember this password, as there is absolutely
	no recovery mechanism.  However, the password can be changed
	later using encfsctl.

	New Encfs Password: 
	Verify Encfs Password: 
$ ls -l b
	total 4
	-rw-rw-r--. 1 jakob jakob    0  2. Jan 13:02 B9ZT7iQhSf0LupxqVCsS2Edg
	-rw-rw-r--. 1 jakob jakob 1264  2. Jan 13:03 ii328JX2cjDCDeNjVQqKGAU1
$ rm b/B9ZT7iQhSf0LupxqVCsS2Edg 
	rm: cannot remove 'b/B9ZT7iQhSf0LupxqVCsS2Edg': Read-only file system
$ fusermount -u b
$ encfs --reverse $PWD/a $PWD/b
	EncFS Password: 
$ rm b/B9ZT7iQhSf0LupxqVCsS2Edg # Works 
```

**Comment by diskmaster23** [comment]
I think you are correct. I think it's a bug with the previous version. I had 1.8.1-3 from the Ubuntu repository installed. It wouldn't allow me to delete files. I upgraded to 1.9.1-3 and this problem cleared up for me.

**Comment by benrubson** [comment]
Solved by #299.

**Comment by benrubson** [comment]
#301 has been merged, can we close this @syaifulnizamyahya @rfjakob ?
Thx !
